benefits update

choldgraf opened the pull request:
adding a benefits page. ping the @2i2c-org/founders in case they have any comments or suggestions for this. I'll plan to merge in a few days if nobody raises objections!

yuvipanda commented on benefits.md:
```diff
@@ -0,0 +1,43 @@
+
+### How to let the team know you're taking off
+
+2i2c uses a Google Calendar to track major events and dates. If you're planning to take time off, as soon as possible **mark these days on your Google Calendar** and **ping the 2i2c team in Slack**.
```
```suggestion
2i2c uses a Google Calendar to track major events and dates. If you're planning to take time off, as soon as possible **mark these days on your Google Calendar** and **tell the rest of the 2i2c team in Slack**.
```

yuvipanda commented on benefits.md:
```diff
@@ -0,0 +1,43 @@
+First and foremost, 2i2c fosters a culture of healthy balance between work and life.
+All 2i2c employees should take the time they need to thrive - this will both make you happier and benefit 2i2c in the long run.
+
+Currently 2i2c does not have a limit on the number of days you may take off each year. We encourage a **don't ask, do tell** policy to time off:
```
I really like https://buffer.com/resources/employees-take-vacation/. Primary problem is that when folks don't know how much vacation others are taking, it causes them to take less time off than they 'should'. 

The primary additions here would be the suggested minimum vacation days policy. I like buffer's example of 15 workdays (~3 weeks). 

I also like the other aspects of their policy, which I've cribbed below.

---
Here are the exact guidelines we sent to the team:

1. We encourage team members to take a minimum of 3 weeks (15 work days) of vacation time throughout the year, in addition to the holidays (bank, religious or otherwise) you choose to observe.
2. There is no maximum vacation recommendation, though there may be more and longer conversations for unique situations in which a teammate might want to take, say, 6–8 weeks off.
3. Buffer is privileged to be a global team; please take time off that’s in line with your country’s standards if it’s generally a bit more than our recommendation.
4. It’s generally great to begin to take time off after you’ve been at Buffer for about 3 months. (If you’re newer to the team and have something already planned, no worries! We’ll take it case by case).
5. This time is separate from holidays that you choose to celebrate, and outside of sick/personal days, bereavement/compassionate leave, and family leave.

yuvipanda replied:
Not sure how to make this interact with the 14day winter holiday though. I also know many people (including myself!) don't like going places or doing traditional things around the winter break. I used to always just trade that for more time in February.

choldgraf replied:
I like the buffer docs on this as well, we should include them as inspiration for this. I suspect that this policy might evolve over time, so I wonder what we can use as an MVP right now and then open issues to discuss the finer points later (since we don't have *any* process right now, it seems like just having something would be an improvement as long as it is not really controversial).

Also just a note that the 2 weeks at the end-of-year thing is just us inheriting an ICSI policy...I'm not sure how that works but my assumption is that we should assume that our policy is constrained by ICSI policy to some degree unless we have an official agreement otherwise? Either way - something we can continue to iterate on

yuvipanda replied:
Yeah, this sounds good to me.

yuvipanda commented on benefits.md:
```diff
@@ -0,0 +1,43 @@
+
+- Let the team know in advance when you'll take time off
+- Give advance notice that roughly scales with how long you'd like to take off (e.g., for just a day or two, let everyone know the week before, to take off several weeks, let us know a month in advance, etc).
+- Try to avoid taking time off during important moments of the academic calendar - this generally means the beginning and end of Fall and Spring semesters.
```
Hmm, thinking about it some more - I think this doesn't apply for everyone. For example, whoever is helping doing just PANGEO stuff. Perhaps this is just a part of 'Let the team know in advance'?

rabernat replied:
This is probably good guidance for anyone in the academic sphere, which includes Pangeo. These are crunch times for research as well.

choldgraf replied:
yeah good point - my main goal here was to try and give an idea that there may be some times of the year when we'd really like people to be "on deck" in stead of on vacation. As an example, Basecamp [uses "CODE" statuses to define when people shouldn't take time off](https://basecamp.com/handbook/08-benefits-and-perks#:~:text=At%20Basecamp%2C%20we%20assign%20a,of%20launching%20a%20new%20product.)...maybe we should do something like this?

yuvipanda reviewed (changes requested):
Thanks for working on this, @choldgraf!

clcarson commented:
I think it could be useful to have a planned and structured review process for posting important policy updates, like I'm guessing this one is. It's hard for me to keep up right now, and it may be for @colliand, @fperez , and maybe others. And I don't know that we've got clarity among ourselves where decision rights lie.

Might we (@choldgraf ?)
- Create/share a list of planned updates
- Maybe have a top-level summary of what the key questions are
- Maybe set up up a process for whole-group structured check-ins with agendas

I don't have experience with digital remote-work environments, so that may be part of my discomfort. But it's also that some of us have heavy demands from our other jobs (teaching? what sabbatical?), and I think we might need to be more direct and structured in dealing with that. Tell me if I'm missing something, though.

choldgraf commented:
Hey @clcarson - thanks for bringing this up. I'll bring conversation around team decisions etc over here: https://github.com/2i2c-org/meta/issues/96

on this issue specifically, at least right now I am just trying to use this PRs to make implicit information explicit, as opposed to making any major changes to our policy. I'm hoping that most of the language here is non-controversial (most of it is just mirroring what ICSI's policy is, which I assumed we *must* abide by, with the exception of respecting home-country holidays). That said, if anything in here is non-controversial then we should figure out what is the minimal non-controversial policy that we can merge (so that there is _some_ official policy) and then discuss how to change it from there

choldgraf commented:
Hey all - I've just pushed some more changes with the following main takeaways:

- Updated the language to use some of Buffer's descriptions about recommended minimum PTO
- Removed language about "Don't ask, do tell" as this is a slightly more radical approach that we should have team alignment on first, and I don't wanna block getting a policy written down on this. Instead I've noted that we use an "Ask the Director" approach for now but plan to re-visit once the team evolves a bit.
- Added a short "inspiration" section at the end w/ links to other policies

choldgraf commented:
Note that when this is merged in, we should delete the content in https://2i2c.org/careers/#salaries-and-benefits and redirect it here. (that text is copy-pasted from ICSI stock text, so it is technically our process right now until we adopt an official 2i2c one, which is what this PR is doing)

choldgraf commented:
Hey all - could we get some 👀 and 👍 or 👎 on this soon? I'm going to post our job posting for the Pangeo position, and while we currently have a salary/benefits page, it is the ICSI stock text and not something that is 2i2c-specific.

choldgraf commented:
Hey all - could we get some 👀 and 👍 or 👎 on this soon? I'm going to post our job posting for the Pangeo position, and while we currently have a salary/benefits page, it is the ICSI stock text and not something that is 2i2c-specific.

yuvipanda commented on benefits.md:
```diff
@@ -0,0 +1,48 @@
+- Currently 2i2c does not have a limit on the number of days you may take off each year. 
+- We recommend that all 2i2c team members **take at least 15 days** (3 weeks) annually (pro-rated if you started work mid-way through the year).
+- Before planning time off, get approval from the Director of 2i2c to make sure there aren't any major dates that we'll need team members for.
+  - In the future, we aim to be more flexible with PTO, but will use this process while the team is very small.
```
I like this explicit call out.

GeorgianaElena reviewed (approved):
Thanks @choldgraf :tada:

GeorgianaElena commented on benefits.md:
```diff
@@ -0,0 +1,48 @@
+
+2i2c is a fully remote team, and as such we recognize holidays based on your home country. You should take the time to be with those around you when your community has time off.
+
+Mark your 2i2c Google Calendar with any nationally-recognized holidays in your country of residence. Use the `HOLIDAY` title for the event on your calendar.
```
This is a great idea :+1:

consideRatio commented on reference.md:
```diff
@@ -32,8 +32,10 @@ Here are some helpful terms that we use at 2i2c.
   Remote teams and open communities need to balance information across team members, and ensure that everyone is on the same page. For this reason, it is recommended to adopt a "single source of truth" for anything important [^sst1]. This is an authoritative source that everyone can look to in order to know the current status and plan for anything we do at 2i2c.
 
+International Computer Science Institute
 ICSI
```
Is either this or the other ordered in the wrong way? I'm not sure how the glossary directive works but perhaps an abbreviation is supposed to come first or last?

choldgraf replied:
I think either one of them become terms that you can include with {term}, the order shouldn't matter

consideRatio replied:
Ah makes sense!

choldgraf commented:
Closing this PR in favor of the following benefits policy draft - please comment there: https://docs.google.com/document/d/1uvfcQUJYZXK0RTLOQ6lx3Lc7JzSbm_p0sieWK6OQ1bQ/edit#
